util/getNodeDownloadSnippet.ts

```diff
@@ -0,0 +1,87 @@
+import dedent from 'dedent';
+
+import type { UserOS } from '@/types/userOS';
+
+export const getNodeDownloadSnippet = (major: number, os: UserOS) => {
+  if (os === 'LINUX' || os === 'MAC') {
+    const platformSnippets = {
+      NVM: dedent`
+        # Installs NVM (Node Version Manager)
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+
+        # Installs Node.js
+        nvm install v${major}
+
+        # Checks that Node is installed
+        node -v
+
+        # Checks your NPM version
+        npm -v`,
+      BREW: dedent`
+        # Installs Brew (macOS/Linux Package Manager)
+        curl -o- https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | bash
+
+        # Installs Node.js
+        brew install node@${major}
+
+        # Checks that Node is installed
+        node -v
+
+        # Checks your NPM version
+        npm -v`,
+      DOCKER: '',
+    };
+
+    if (os === 'MAC') {
+      platformSnippets.DOCKER = dedent`
+        # Installs Brew (macOS Package Manager)
+        curl -o- https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | bash
+
+        # Installs Docker Desktop
+        brew install docker --cask
+
+        # Pull Node.js Docker Image
+        docker pull node:${major}-${major >= 4 ? 'alpine' : 'slim'}
```

Comment:
tiny nit: i know most people now say to use image SHAs, but that might be hard to communicate